src/main/java/taub/citi/Station.java

```diff
@@ -0,0 +1,85 @@
+package taub.citi;
+
+public class Station
+{
+    //CHECKSTYLE:OFF
+    public String station_type;
```

Comment:
Only include the variables you need to use. Not everything.

Reply:
What if I am not sure what I need yet? Can I keep it in for now?

Reply:
Ok...

Reply:
I fixed the gitignore and the other comments. Am I able to merge my pr?